internal/provider/resource_gitlab_group_saml_link_test.go

```diff
@@ -0,0 +1,79 @@
+//go:build acceptance
+// +build acceptance
+
+package provider
+
+import (
+	"fmt"
+	"testing"
+
+	"github.com/hashicorp/terraform-plugin-sdk/v2/helper/resource"
+	"github.com/hashicorp/terraform-plugin-sdk/v2/terraform"
+)
+
+func TestAccGitlabGroupSamlLink_basic(t *testing.T) {
+	testAccRequiresAtLeast(t, "15.3")
+
+	testGroup := testAccCreateGroups(t, 1)[0]
+
+	resource.ParallelTest(t, resource.TestCase{
+		ProviderFactories: providerFactories,
+		CheckDestroy:      testAccCheckGitlabGroupSamlLinkDestroy,
+		Steps: []resource.TestStep{
+
+			// Create a group SAML link as a developer
+			{
+				Config: fmt.Sprintf(`
+					resource "gitlab_group_saml_link" "this" {
+						group   		= "%d"
+						access_level 	= "Developer"
```

Comment:
Goes together with the other change:

```suggestion
						access_level 	= "developer"
```